assets/main.js

```diff
@@ -26,6 +26,15 @@ function toggleAboutVisibility() {
 }
 
 
+if (window.matchMedia("(max-media: 700px)").matches) {
```

Comment:
It would make more sense if the conditional was inside the function, no?
Otherwise if the function is called from desktop we'll get a "function not defined" type error.

Reply:
I think I remember putting it inside the function and it not working. But, I'll try it inside the function and get back to you.

Reply:
Yeah, it doesn't work if the conditional is inside the function.